images/openstack/Dockerfile.ci

```diff
@@ -1,4 +1,4 @@
-# This Dockerfile is used by CI to test using OpenShift Installer against an OpenStack cloud.
+yum clean all && rm -rf /var/cache/yum/*# This Dockerfile is used by CI to test using OpenShift Installer against an OpenStack cloud.
```

Comment:
I think this is wrong

Reply:
That should just be a forced clean up of any yum cache. Do you think that yum cache should not be cleaned here?

Reply:
Isn't that already happening at https://github.com/openshift/installer/blob/master/images/openstack/Dockerfile.ci#L21?

Reply:
got it this was not what I was thinking it was. Thank you ! This was somehow pasted there.